src/platform/guiqt.cpp

```diff
@@ -27,6 +27,9 @@
 #include <QSettings>
 #include <QTimer>
 
+#ifdef WIN32
+#    include <windows.h>
```

Comment:
What symbol is windows.h providing?

Reply:
Essentially allows this:

https://github.com/solvespace/solvespace/blob/fdb80e3ea7a47fbfe73fce308fd9d65faab5aa58/src/platform/guiqt.cpp#L973

to compile.